megamek/src/megamek/client/ui/swing/CommonSettingsDialog.java

```diff
@@ -1202,10 +1224,6 @@ public void itemStateChanged(ItemEvent event) {
             stampFormatLabel.setEnabled(stampFilenames.isSelected());
         } else if (source.equals(fovInsideEnabled)) {
             guip.setFovHighlight(fovInsideEnabled.isSelected());
-            if ((clientgui != null) && (clientgui.bv != null)) {
-                clientgui.bv.clearHexImageCache();
-                clientgui.bv.repaint();
-            }
```

Comment:
Just checking, but we don't need to do either of these to properly update the board with the newly changed state?

Reply:
No, these lines are actually nonsense I may have put in myself a few years ago. The boardview does preferencechange-listening and can update when such an event is fired. It just needs the right checks for what pref has changed to update accordingly. It could even update without checking what has changed but I think that's bad form, as all hexes have to be redrawn etc.